### ISSUE TITLE
Footer items turn invisible on hover

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/models#54]
When the user hovers over green hyperlinks in the footer of [models.accordproject.org](models.accordproject.org) and [templates.accordproject.org](models.accordproject.org) such as "ACCORD PROJECT" and "APACHE-2.0", the text turns black, rendering it nearly invisible. Change the hover effect so that the hyperlinks turns white on hover. 

Make sure this does not change the hover effect of all of the other hyperlinks on the page that aren't in the footer.

![image](https://user-images.githubusercontent.com/40569793/66155432-9a490f80-e5ed-11e9-905e-64600b38d1eb.png)

**Comment by schmitty890** [comment]
Looks awesome! I can look into this!

**Comment by Michael-Grover** [comment]
@schmitty890 great, thank you! let me know if you need anything
